ad9959/src/lib.rs

```diff
@@ -568,6 +609,58 @@ impl ProfileSerializer {
         }
     }
 
+    /// Update a number of channels with fully defined profile settings.
+    ///
+    /// # Args
+    /// * `channels` - A set of channels to apply the configuration to.
+    /// * `profile` - The complete DDS profile, which defines the frequency tuning word,
+    ///   amplitude control register & the phase offset word of the channels. Note that the ACR
+    ///   should be stored in the 3 LSB of the word. If amplitude scaling is to be used, the
+    ///   "Amplitude multiplier enable" bit must be set.
+    #[inline]
+    pub fn update_channels_with_profile(
```

Comment:
There is no point in duplicating the code in `update_channels`.
This method is not needed.